### PR TITLE
Get HCAv2 ready for UAT as a beta app

### PR DESCRIPTION
**Pull request by erikphansen**
## Description
We are going to be conducting UAT of the changes to the HCA as a beta app at `va.gov/hca2`. Currently all the relevant code changes are on staging, hidden behind the `isProduction()` environment flag.

NOTE: The vast majority of this PR is due to duplicating the entire existing `hca` app as `hca2` and making a few small changes to that duplicated code.

This change does the following:
- duplicates the existing `hca` app as a `hca2` app
- creates a new entryname, `hca2`, for this app
- hides the entire new app behind the BetaApp component

## Testing done
Local

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/hca2/HealthCareApp.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import RoutedSavableApp from '../../platform/forms/save-in-progress/RoutedSavableApp';
+import formConfig from './config/form';
+
+import BetaApp from 'applications/beta-enrollment/containers/BetaApp';
+import { features } from 'applications/beta-enrollment/routes';
+
+export default function HealthCareEntry({ location, children }) {
+  return (
+    <BetaApp featureName={features.hca2} redirect="/beta-enrollment/hca/">
```
Comment (erikphansen) [inline src/applications/hca2/HealthCareApp.jsx]: This should wrap the entire HCA app in the `BetaApp` component.

**Review by ncksllvn — APPROVED** [review]

**Review by rianfowler — APPROVED** [review]
maybe call it hca-v2-beta

**Comment by erikphansen** [comment]
@rianfowler as the entryName?

**Comment by rianfowler** [comment]
yeah and maybe the folder name- `hca2` is a little terse.

**Pull request merged**